src/main/java/com/greenfox/poker/service/UserService.java

```diff
@@ -88,8 +94,12 @@ public boolean isUserExistsInDB(long id) {
     List<PokerUser> topTenList = pokerUserRepo.findTop10ByOrderByChipsDesc();
     List<PokerUserDTO> topTenDTO = new ArrayList<>();
```

Comment:
Top ten what? :)
Top ten list means that you have 10 lists.
And what does the DTO represent?
Give better names here.